lib/Caxy/HtmlDiff/HtmlDiff.php

```diff
@@ -534,7 +534,7 @@ protected function findIsolatedDiffTagsInOld($operation, $posInNew)
     protected function insertTag($tag, $cssClass, &$words)
     {
         while (true) {
-            if (!is_countable($words) || count($words) == 0) {
+            if (!is_array($words) || count($words) == 0) {
```

Comment:
Hi @daum could you please to strict type checking here? (using `===`) and rebase this PR on the latest `master`